src/v2/Components/ArtworkFilter/ArtworkFilters/__tests__/SizeFilter2.rtl.jest.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { ReactElement } from "react"
+import {
+  render as originalRender,
+  RenderOptions,
+  screen,
+} from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+
+import {
+  ArtworkFilterContextProps,
+  ArtworkFilterContextProvider,
+  useArtworkFilterContext,
+} from "v2/Components/ArtworkFilter/ArtworkFilterContext"
+import { SizeFilter2 } from "../SizeFilter2"
+
+const render = (ui: ReactElement, options: RenderOptions = {}) =>
+  originalRender(ui, { wrapper: Wrapper, ...options })
+
+const Wrapper: React.FC = ({ children }) => {
+  return (
+    <ArtworkFilterContextProvider>
+      <ClearAllButton />
+      {children}
+      <ArtworkFilterContextInspector />
+    </ArtworkFilterContextProvider>
+  )
+}
+
+const ClearAllButton: React.FC = () => {
+  const artworkFilterContext = useArtworkFilterContext()
+
+  return (
+    <button onClick={() => artworkFilterContext.resetFilters()}>
+      Clear all
+    </button>
+  )
+}
+
+const ArtworkFilterContextInspector: React.FC = () => {
+  const artworkFilterContext = useArtworkFilterContext()
+
+  return (
+    <aside data-testid="artwork-filter-context-inspector">
+      {JSON.stringify(artworkFilterContext, null, 2)}
+    </aside>
+  )
+}
+
+const currentContext = (): ArtworkFilterContextProps => {
+  let contextInspector: HTMLElement
+  try {
+    contextInspector = screen.getByTestId("artwork-filter-context-inspector")
+  } catch (error) {
+    if (error.name === "TestingLibraryElementError")
+      throw new Error(
+        `The currentContext() helper function requires an <ArtworkFilterContextInspector /> to be mounted in the current DOM.`
+      )
+  }
+  return JSON.parse(contextInspector!.textContent!)
+}
+
+describe("SizeFilter2", () => {
+  it("toggles custom input render", () => {
+    render(<SizeFilter2 expanded />)
+
+    // note: an <input type="number" /> has a role of 'spinbutton'
+    expect(screen.queryByRole("spinbutton")).not.toBeInTheDocument()
```

Comment:
The inputs in this UI use `<input type="number" />`:

<img width=200 src="https://user-images.githubusercontent.com/140521/119671570-e02c9a00-be07-11eb-84e0-b98d8d948a0e.gif" />

TIL these have an aria role of `spinbutton`, so that's the RTL way of finding these elements.